package.json

```diff
@@ -26,6 +26,18 @@
   },
   "homepage": "https://github.com/isaacpitwa/awesome-books-es6#readme",
   "dependencies": {
-    "esm": "^3.2.25"
-  }
+    "esm": "^3.2.25",
```

Comment:
- [ ] I cannot start your project because of some errors in your js file. Kindly fix it before asking for another code review
![Screenshot from 2022-03-23 00-20-12](https://user-images.githubusercontent.com/62705947/159592192-cbe2d3fe-7df7-44ee-bf4c-ce55a61bd9a4.png)